connect-examples/oauth/node/index.js

```diff
@@ -0,0 +1,80 @@
+/* This sample demonstrates a bare-bones implementation of the Square Connect OAuth flow in Node.js:
+    1. A merchant clicks the authorization link served by the root path (http://localhost:8000/sandbox_request_token)
+    2. The merchant signs in to Square and submits the Permissions form. Note that if the merchant
+        is already signed in to Square, and if the merchant has already authorized your application,
+        the OAuth flow automatically proceeds to the next step without presenting the Permissions form.
+    3. Square sends a request to your application's Redirect URL
+        (which should be set to http://localhost:8000/sandbox_callback on your application dashboard)
+    4. The server extracts the authorization code provided in Square's request and passes it
+        along to the Obtain Token endpoint.
+    5. The Obtain Token endpoint returns an access token your application can use in subsequent requests
+        to the Connect API.
+    This sample requires the following dependencies:
+        express (https://expressjs.com/)
+*/
+
+
+const express = require('express');
+const SquareConnect = require('square-connect');
+const app = express();
+
+const port = process.env.PORT || "8000";
+const messages = require('./sandbox-messages')
+const config = require('./config.json')
+
+// Configure Square defcault client
+const defaultClient = SquareConnect.ApiClient.instance
+defaultClient.basePath = config.SQ_SANDBOX_BASEURL
+
+// Configure Square OAuth API instance
+const oauthInstance = new SquareConnect.OAuthApi();
+const scopes = [] // INCLUDE PERMISSIONS YOU WANT YOUR SELLER TO GRANT YOUR APPLICATION
+
+// Serves the link that merchants click to authorize your application
```

Comment:
please use the consistent comment style.

the node_order-payments example has the best style:

```
/**
 * Matches: GET /checkout/choose-delivery-pickup/
 *
 * Description:
 *  Render the page for customer to submit delivery or pickup informaiton
 *  to update the corresponding order (order_id)
 *
 * Query Parameters:
 *  order_id: Id of the order to be updated
 *  location_id: Id of the location that the order belongs to
 */

```